incubator/kafka-connect/README.md

```diff
@@ -0,0 +1,132 @@
+# kafka-connect
```

Comment:
It would be great to aim straight for the stable repo - however that would mean that the dependencies should be in stable too so maybe not realistic at this point in time

Also please add a .htlmignore file

See RBAC-bestpractices: https://github.com/helm/helm/blob/master/docs/chart_best_practices/rbac.md

Reply:
@davidkarlsen -- what are you looking for re: RBAC? The chart doesnt need anything other than default permissions since it doesnt interact with the cluster other than if kafka is running in kubernetes. Is the policy just to have a specific RBAC role created by default in all new charts?